docs/CONFIG.md

```diff
@@ -1422,6 +1422,46 @@ DisableRateLimiting = false # Default
 ```
 DisableRateLimiting skips ratelimiting on asset requests.
 
+## Tracing
+```toml
+[Tracing]
+Enabled = false # Default
+CollectorTarget = "localhost:4317" # Example
+NodeID = "NodeID" # Example
+```
+
+
+### Enabled
+```toml
+Enabled = false # Default
+```
+Enabled turns trace collection on or off. On requires an OTEL Tracing Collector.
+
+### CollectorTarget
+```toml
+CollectorTarget = "localhost:4317" # Example
+```
+CollectorTarget is the logical address of the OTEL Tracing Collector.
+
+### NodeID
+```toml
+NodeID = "NodeID" # Example
+```
+NodeID is an unique name for this node relative to any other node traces are collected for.
+
+## Tracing.Attributes
+```toml
+[Tracing.Attributes]
+user = "test" # Example
+```
+Tracing.Attributes are user specified key-value pairs to associate in the context of the traces
+
+### user
+```toml
+user = "test" # Example
```

Comment:
i find the example of `user` confusing because it's static and the isn't an obviously meaningful map from a node to user. perhaps something like `env = test ` is a better or some other tag that delineates one node from another.

said differently in a tracing system i would expect the system to derive a user and populate the trace as needed - if there is only one user than it's not useful and if there are multiple then it has to dynamic

Reply:
Agreed, I can update to `env`